### PR TITLE
Update officer roles

### PR DESCRIPTION
**Pull request by tsadler1988**
Update board page to reflect changes to officers.

### CODE REVIEW

**Review by rrrutledge — APPROVED** [review]
Wow!  That’s a lot to have to do all the languages.

File: content/en/about/board/_index.md [hunk 1]
```diff
@@ -82,14 +82,6 @@ Previously, Russ founded and led the Developer Collaboration effort at Nike.
 Russ's drive and passion is to enable all software engineers to achieve incredible technical and business throughput via quality tooling and streamlined work process.
 {{< /board-member >}}
 
-{{< board-member name="Silona Bonewald" title="Treasurer" image="/images/about/Silona_Bonewald.jpeg" >}}
```
Comment (spier) [inline content/en/about/board/_index.md:85]: Should we move Silone and max to the "Past Directors and Officers"?
I don't know if all translations have this section but EN does.

**Review by yuhattor — APPROVED** [review]
LGTM!

File: content/en/about/board/_index.md (hunk 1 above)
Comment (tsadler1988) [reply content/en/about/board/_index.md:85]: Yes we should, good spot @spier!

Comment (tsadler1988) [reply content/en/about/board/_index.md:85]: Spotted a few other issues across languages. Fixed what I can and raised https://github.com/InnerSourceCommons/innersourcecommons.org/issues/705 to address the rest.

**Review by rrrutledge — APPROVED** [review]

**Pull request merged**